app/src/main/java/de/danoeh/antennapod/activity/VideoplayerActivity.java

```diff
@@ -480,4 +484,96 @@ void stop() {
 
     }
 
+
+    //Hardware keyboard support
+    @Override
+    public boolean onKeyUp(int keyCode, KeyEvent event) {
+        AudioManager audioManager = (AudioManager) getSystemService(AUDIO_SERVICE);
+        Float videoPercent = null;
+
+        switch (keyCode) {
+            case KeyEvent.KEYCODE_ENTER: //Fallthrough
+            case KeyEvent.KEYCODE_SPACE:
+                //Play/Pause
+                onPlayPause();
+                toggleVideoControlsVisibility();
+                break;
+            case KeyEvent.KEYCODE_DPAD_LEFT:
+                //Go Back
+                onRewind();
```

Comment:
For double-tapping forward and backward, we already have a nice little animation. Could you try to add it here, too?